code/game/objects/structures/__structure.dm

```diff
@@ -40,15 +39,15 @@
 		fluid_update(TRUE)
 
 /obj/structure/get_examined_damage_string(health_ratio)
-	if(maxhealth == -1)
+	if(max_health == -1)
```

Comment:
ITEM_HEALTH_NO_DAMAGE?

Reply:
I'd rather keep the focus to moving the vars in this PR. I don't want to give the impression that structures actually like are guaranteed to handle that special value at all, cause they don't yet. Most of their damage handling is tailor made.

Reply:
They all can, and they will use a getter and use the proper overridable procs.

I'm trying to make bite-sized PRs that don't fundamentally change the existing logic, to reduce conflicts and to make them faster to review. Also, if I start messing with that, I know I'm gonna start changing way too many things XD

Honestly, I probably should have left the procs out for this PR.. I could cherry pick the commit out of this PR.